cub/cub/util_type.cuh

```diff
@@ -885,7 +885,7 @@ struct KeyValuePair<K, V, false, true>
  * \brief A wrapper for passing simple static arrays as kernel parameters
  */
 template <typename T, int COUNT>
-struct ArrayWrapper
+struct CUB_DEPRECATED ArrayWrapper
```

Comment:
>  we tried to postpone all deprecations until a major release to avoid source compatibility issues on minor CTK updates. Is it still the recommended way or we can relax this requirement given the small ArrayWrapper user base?

Adding new deprecation warnings in a minor release is fair game as far as I'm concerned. We should just make sure to include in the emitted warning to tell people how to turn the warning off.

Reply:
@gevtushenko yes currently we cannot build any `\deprecated` and `@deprecated` doxygen comment

Reply:
@miscco what does this mean for this PR now? Can I add a `@deprecated bla bla` comment now, or not?

Reply:
It will break our docs soon, so I believe you should do `deprecated bla bla` so that it is there but is not picked up by doxygen 

That way when we fix this it will be easy to bring in